queries.sql

```diff
@@ -8,3 +8,50 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = True;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+/* transaction one */
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT species FROM animals;
+ROLLBACK;
+SELECT species FROM animals;
+
+/* transaction two */
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE name NOT LIKE '%mon';
```

Comment:
## Optional
- [ ] This should update the animal's table by setting the species column to `pokemon` for all animals that don't have species already set. I suggest that you make it `WHERE species IS NULL;` instead of `WHERE name NOT LIKE '%mon';`

Reply:
Noted with thanks